TEMPLATES/mega-linter.yml

```diff
@@ -40,129 +120,227 @@ jobs:
     name: MegaLinter
     runs-on: ubuntu-latest
 
-    # Give the default GITHUB_TOKEN write permission to commit and push, comment
-    # issues & post new PR; remove the ones you do not need
+    ##
+    # This job's `GITHUB_TOKEN` or your added `PAT` must have the following
+    # permissions enabled to `write` for MegaLinter to function.
+    #
+    # Always aim to provide as few permissions as possible for personal access
+    # tokens. By default, all permissions are set to read only. Write
```

Comment:
Is it read-only or read only in English?

Reply:
Either is acceptable - it's preference. I think read-only would likely make more sense, in this regard.

Reply:
read-only then